powerauth-backend-tests/pom.xml

```diff
@@ -86,6 +86,11 @@
             <artifactId>powerauth-restful-security-spring</artifactId>
             <version>1.3.0-SNAPSHOT</version>
```

Comment:
Shouldn't this version be 1.4.0?

Reply:
Yes, we should switch to `1.4.0-SNAPSHOT`.